google-cloud-spanner/src/main/java/com/google/cloud/spanner/Value.java

```diff
@@ -823,6 +871,13 @@ public String getPgJsonb() {
     throw new UnsupportedOperationException("Not implemented");
   }
 
+  /**
+   * Returns the value of a {@code PG_OID}-typed instance.
+   *
+   * @throws IllegalStateException if {@code isNull()} or the value is not of the expected type
+   */
+  public abstract long getPgOid();
```

Comment:
As we expect users to call `ResultSet#getInt64(..)` for OID, we should do the same here. So remove this method and the array equivalent, and make sure that the `getInt64(..)` here also works for OID.

Reply:
Done.